python_modules/dagster/dagster/_core/execution/context/compute.py

```diff
@@ -607,7 +972,61 @@ def asset_partitions_def_for_input(self, input_name: str) -> PartitionsDefinitio
 
     @public
     def asset_partition_keys_for_output(self, output_name: str = "result") -> Sequence[str]:
-        """Returns a list of the partition keys for the given output."""
+        """Returns a list of the partition keys for the given output.
+
+        If you want to write your asset to support running a backfill of several partitions in a single run,
+        you can use ``asset_partition_keys_for_output`` to get all of the partitions being materialized
+        by the backfill.
+
+        Args:
+            output_name (str): The name of the output produced by the asset to get the keys for.
+                If not provided for assets defined with the `@asset` decorator, the name of the output
+                will be automatically provided.
+
+        Examples:
+            .. code-block:: python
+
+                partitions_def = DailyPartitionsDefinition("2023-08-20")
```

Comment:
update example to use a Static partition, since this method was intended for non-time based partitions